.gitignore

```diff
@@ -4,14 +4,17 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+# openscap vulnerability report
+vuln.html
+
 # Binaries for programs and plugins
 *.exe
 *.exe~
 *.dll
 *.so
 *.dylib
 bin
-preflight
+openshift-preflight
```

Comment:
I think adding `openshift-preflight` is fine, but I think the Makefile still gives us a `preflight` binary. Let's keep both?

Reply:
+1 to @komish  comment. The binary call should just be `preflight <subcommand>`.  We still need to .gitignore `preflight`

Reply:
Not sure why this is happening, but running `go build` on my local (via Vagrant) creates a binary named `openshift-preflight`, and not `preflight`. I have not changed the Makefile and the `-o preflight` is set properly.

Reply:
Are you running `go build` or `make build`? I think the former takes the parent directory name into consideration but doesn't use the make file. The latter does whatever is in the make file (which should spit out `preflight` as the binary).

But either way - keep both in, just in case. 👍

Reply:
Nice! Did not realize we are using `make`. I need to start a developer guide :)